include/oneapi/dpl/internal/exclusive_scan_by_segment_impl.h

```diff
@@ -91,18 +92,27 @@ exclusive_scan_by_segment_impl(Policy&& policy, InputIterator1 first1, InputIter
 }
 
 #if _ONEDPL_BACKEND_SYCL
+
```

Comment:
Is that something clang-format is suggesting? or is that just redundant change?

Reply:
It was an unintended change. I will remove this empty line.